### ISSUE TITLE
Add to Safari Reading list broken

### ISSUE THREAD

**Issue by rgunnyon** [ViennaRSS/vienna-rss#595]
Since 3.1.0
2016-03-20 11:25:34.175 AM Vienna[927]: An exception was thrown during execution of an NSScriptCommand...
2016-03-20 11:25:34.176 AM Vienna[927]: [<Folder 0x7fad69fc39f0> valueForUndefinedKey:]: this class is not key value coding-compliant for the key articles.

Running on OS X 10.11

**Comment by josh64x2** [comment]
`articles` ivar has been removed from `Folder` class causing this issue.

**Comment by barijaona** [comment]
As the article is already selected, the cache is ready, but there is no enumerator for a NSCache. We'll have to enumerate manually :
````
 for (id obj in cachedGuids.reverseObjectEnumerator.allObjects)
 {
    Article * article = [cachedArticles objectForKey:(NSString *)obj];
 ...

**Comment by josh64x2** [comment]
@barijaona shall i make this change quickly in time for 3.1.2?

**Comment by barijaona** [comment]
Fixed in 3.1.2
